src/ado2gh/Commands/GenerateMannequinCsvCommand.cs

```diff
@@ -0,0 +1,107 @@
+        {
+            _log = log;
+            _githubApiFactory = githubApiFactory;
+
+            Description = "Generates a CSV with unreclaimed mannequins to reclaim them in bulk.";
+            IsHidden = true;
+
+            var githubOrgOption = new Option<string>("--github-org")
+            {
+                IsRequired = true,
+                Description = "Uses GH_PAT env variable."
+            };
+
+            var outputOption = new Option<string>("--output", () => "./mannequins.csv")
+            {
+                IsRequired = false,
+                Description = "Output filename. Default value mannequins.csv"
+            };
+
+            var includeReclaimedOption = new Option("--include-reclaimed")
+            {
+                IsRequired = false,
+                Description = "Map the user even if it was previously mapped"
+            };
+
+            var verbose = new Option("--verbose")
+            {
+                IsRequired = false
+            };
+
+            var githubPat = new Option<string>("--github-pat")
+            {
+                IsRequired = false
+            };
+
+            AddOption(githubOrgOption);
+            AddOption(outputOption);
+            AddOption(includeReclaimedOption);
+            AddOption(githubPat);
+            AddOption(verbose);
+
+            Handler = CommandHandler.Create<string, string, bool, string, bool>(Invoke);
+        }
+
+        public async Task Invoke(
+          string githubOrg,
+          string output,
+          bool includeReclaimed = false,
+          string githubPat = null,
+          bool verbose = false)
+        {
+            _log.Verbose = verbose;
+
+            _log.LogInformation("Generating CSV...");
+
+            _log.LogInformation($"GITHUB ORG: {githubOrg}");
+            if (githubPat is not null)
+            {
+                _log.LogInformation("GITHUB PAT: ***");
+            }
+            _log.LogInformation($"FILE: {output}");
+            if (includeReclaimed)
+            {
+                _log.LogInformation("INCLUDING RECLAIMED");
+            }
+
+            var githubApi = _githubApiFactory.Create(personalAccessToken: githubPat);
+
+            var githubOrgId = await githubApi.GetOrganizationId(githubOrg);
+            _log.LogInformation($"    Organization Id: {githubOrgId}");
+
+            var mannequins = await githubApi.GetMannequins(githubOrgId);
+
+            _log.LogInformation($"    # Mannequins Found: {mannequins.Count()}");
+
+            var numberMannequins = 0;
+            var contents = new StringBuilder().AppendLine("login,claimantlogin");
+            foreach (var mannequin in mannequins.Where(m => includeReclaimed || m.MappedUser is null))
+            {
+                numberMannequins++;
+                contents.AppendLine($"{mannequin.Login},{mannequin.MappedUser?.Login}");
+            }
+
+            await WriteToFile(output, contents.ToString());
```

Comment:
```suggestion
            await WriteToFile(output.FullName, contents.ToString());
```

If we change it to FileInfo I think that's what we need to do to get the path